.mage/js_sdk.go

```diff
@@ -90,6 +90,36 @@ func (k JsSDK) Watch() error {
 	return yarn("run", "build:watch")
 }
 
+// Fmt formats all js files.
+func (k JsSDK) Fmt() error {
+	mg.Deps(JsSDK.Deps)
+
+	if mg.Verbose() {
+		fmt.Println("Running prettier on sdk .js files")
+	}
+	yarn, err := k.yarn()
+	if err != nil {
+		return err
+	}
+
+	return yarn("run", "fmt")
```

Comment:
Why not define `func (k JsSDK) runYarn(string) error` and do
```go
return k.yarnRun(<cmd>) // <cmd> would be "fmt" here, for example
```

instead of the whole
```go
	yarn, err := k.yarn()
	if err != nil {
		return err
	}

	return yarn("run", <cmd>)
```
block in most of these methods?

Reply:
not in this pr. we will revisit this part when publishing the sdk to the npm registry.

Reply:
Can you add a TODO with an issue reference?

Reply:
no. your proposed improvement has nothing to with this specific pr. `Lint()` and `Fmt()` follow the structure that was implemented before.

see https://github.com/TheThingsNetwork/lorawan-stack/issues/30#issuecomment-522974595